src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const listEl = document.querySelector('ul');
```

Comment:
It's a good practice to check if the element was actually found before using it. If there is no 'ul' element on the page, the script will throw an error at line 4.